Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -8695,6 +8793,41 @@ def register_signal_handler_profiling_dump(signal_type=None, profiling_dump_rows
         demisto.info('Not a Linux or Mac OS, profiling using a signal is not supported.')
 
 
+def shorten_string_for_printing(source_string, max_length):
+    """
+    Function that removes the middle of a long str, for printint or logging.
+    If needed, it will replace the middle with '...',
+    e.g. shorten_string_for_printing('123456789', 9) -> '123456789'
+    e.g. shorten_string_for_printing('1234567890', 9) -> 'abc...890'
+    e.g. shorten_string_for_printing('123456789012', 10) -> '1234...012'
```

Comment:
Have you considered using `doctest`?
see: https://docs.python.org/2.7/library/doctest.html

Reply:
https://github.com/demisto/content/pull/16776/commits/f35485946d5771b446a4662020e0e6464986ea11